networkx/algorithms/traversal/tests/test_dfs.py

```diff
@@ -141,12 +173,79 @@ def test_dls_edges(self):
         edges = nx.dfs_edges(self.G, source=9, depth_limit=4)
         assert list(edges) == [(9, 8), (8, 7), (7, 2), (2, 1), (2, 3), (9, 10)]
 
-    def test_dls_labeled_edges(self):
+    def test_dls_labeled_edges_depth_1(self):
         edges = list(nx.dfs_labeled_edges(self.G, source=5, depth_limit=1))
         forward = [(u, v) for (u, v, d) in edges if d == "forward"]
         assert forward == [(5, 5), (5, 4), (5, 6)]
+        # Note: reverse-depth_limit edge types were not reported before gh-6239
```

Comment:
```suggestion
        # Note: reverse-depth_limit edge types were not reported before gh-6240
```

Just a tiny nit pick, #6239 is the issue, 6240 is where this behavior will be added.